### PR TITLE
#149072615 - Fix Heroku Deployment Issue

### PR DESCRIPTION
**Pull request by andela-uofoegbu**
#### What does this PR do?
Ensures the landing page displays properly when a user navigates to https://learning-map-stage.herokuapp.com/
#### Description of Task to be completed?
Fix the Heroku deployment issue
#### How should this be manually tested?
Navigate to https://learning-map-stage.herokuapp.com/
#### Any background context you want to provide?
Previously navigating to the learning map URL leads to an error page, doesn't display homepage on load
#### What are the relevant pivotal tracker stories?
[#149072615](https://www.pivotaltracker.com/n/projects/2059149/stories/149072615)
#### Screenshots (if appropriate)
screen shot 2017-07-19 at 17 19 04

#### Questions:
N/A

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,46 @@
+0 info it worked if it ends with ok
```
Comment (BolajiOlajide) [inline npm-debug.log]: This is a debug log, you're not supposed to push this. 
You can add it to the `.gitignore` file.

File: package.json [hunk 2]
```diff
@@ -77,6 +74,9 @@
...
     "style-loader": "^0.18.2",
     "styled-components": "^2.1.1",
-    "webpack": "^3.0.0"
+    "webpack": "^3.0.0",
+    "webpack-dev-middleware": "^1.11.0",
+    "webpack-hot-middleware": "^2.18.1",
+    "winston": "^2.3.1"
   }
-}
+}
```
Comment (BolajiOlajide) [inline package.json]: an extra line is needed here

**Comment by BolajiOlajide** [comment]
@andela-kakpobome do you think the `dist/` folder is still meant to be here since we've ignored it?

**Comment by andela-uofoegbu** [comment]
@BolajiOlajide Removed the log file, updated the .gitignore file and put an end of line space but for some reason it's not reflecting on github

File: .gitignore [hunk 3]
```diff
@@ -3,3 +3,5 @@ server-transpiled.js
 .env
 .DS_Store
 .idea/
+dist/
+npm-debug.log
```
Comment (BolajiOlajide) [inline .gitignore]: A new line needed here.

Comment (andela-uofoegbu) [inline .gitignore]: done

**Comment by BolajiOlajide** [comment]
LGTM @andela-kakpobome 
If you can confirm the inclusion of the `dist/` directory then i think this is good.

**Comment by andela-kakpobome** [comment]
@andela-uofoegbu @BolajiOlajide it's a generated directory, and so should not be included.

File: src/pages/home/home.scss [hunk 4]
```diff
@@ -84,7 +84,7 @@ html, body, #app {
 }
 
 #logo-bar {
-  background-image: url("../../../assets/images/andela-logo.jpg");
+  background-image: url("../../../assets/image/andela-logo.jpg");
```
Comment (andela-kakpobome) [inline src/pages/home/home.scss]: #### Nitpick

Change this back to images

**Pull request closed without merging**